classes/controllers/personal_deadlines_controller.php

```diff
@@ -168,22 +174,23 @@ protected function get_personal_deadline(){
      * @throws \moodle_exception
      * @throws \require_login_exception
      */
-    public function insert_update($time){
+    public function insert_update($time) {
         global $USER;
         if (!$this->validated($time)) {
             return [
                 'error' => 1,
                 'message' => 'The new deadline you chose has already passed. Please select appropriate deadline'
```

Comment:
another one that should be a language string

Reply:
I opened CTP-3715 for this and the other one mentioned